src/set_theory/surreal.lean

```diff
@@ -269,28 +433,168 @@ instance : has_lt surreal   := ⟨lt⟩
 instance : has_add surreal  := ⟨add⟩
 instance : has_neg surreal  := ⟨neg⟩
 
-instance : ordered_add_comm_group surreal :=
+instance : add_comm_group surreal :=
 { add               := (+),
   add_assoc         := by { rintros ⟨_⟩ ⟨_⟩ ⟨_⟩, exact quotient.sound add_assoc_equiv },
   zero              := 0,
   zero_add          := by { rintros ⟨_⟩, exact quotient.sound (pgame.zero_add_equiv _) },
   add_zero          := by { rintros ⟨_⟩, exact quotient.sound (pgame.add_zero_equiv _) },
   neg               := has_neg.neg,
   add_left_neg      := by { rintros ⟨_⟩, exact quotient.sound pgame.add_left_neg_equiv },
-  add_comm          := by { rintros ⟨_⟩ ⟨_⟩, exact quotient.sound pgame.add_comm_equiv },
-  le                := (≤),
+  add_comm          := by { rintros ⟨_⟩ ⟨_⟩, exact quotient.sound pgame.add_comm_equiv }, }
+
+instance : ordered_add_comm_group surreal :=
+{ le                := (≤),
   lt                := (<),
   le_refl           := by { rintros ⟨_⟩, refl },
   le_trans          := by { rintros ⟨_⟩ ⟨_⟩ ⟨_⟩, exact pgame.le_trans },
   lt_iff_le_not_le  := by { rintros ⟨_, ox⟩ ⟨_, oy⟩, exact pgame.lt_iff_le_not_le ox oy },
   le_antisymm       := by { rintros ⟨_⟩ ⟨_⟩ h₁ h₂, exact quotient.sound ⟨h₁, h₂⟩ },
-  add_le_add_left   := by { rintros ⟨_⟩ ⟨_⟩ hx ⟨_⟩, exact pgame.add_le_add_left hx } }
+  add_le_add_left   := by { rintros ⟨_⟩ ⟨_⟩ hx ⟨_⟩, exact pgame.add_le_add_left hx },
+  .. surreal.add_comm_group }
 
 noncomputable instance : linear_ordered_add_comm_group surreal :=
-{ le_total := by rintro ⟨⟨x, ox⟩⟩ ⟨⟨y, oy⟩⟩; classical; exact
-    or_iff_not_imp_left.2 (λ h, le_of_lt oy ox (pgame.not_le.1 h)),
+{ le_total     := by rintro ⟨⟨x, ox⟩⟩ ⟨⟨y, oy⟩⟩; classical;
+                     exact or_iff_not_imp_left.2 (λ h, le_of_lt oy ox (pgame.not_le.1 h)),
   decidable_le := classical.dec_rel _,
-  ..surreal.ordered_add_comm_group }
+  .. surreal.ordered_add_comm_group }
+
+lemma nonneg_of_nsmul_pos (m : ℕ) {x : surreal} (hx : 0 < x) : 0 ≤ m • x :=
+begin
+  induction m with m hm,
+  { simp only [le_refl, zero_smul] },
+  { rw [succ_nsmul x m],
+    exact add_nonneg (le_of_lt hx) hm }
+end
```

Comment:
This lemma is true (using the same proof) of all ordered_groups, not just `surreal`:
```lean
lemma nsmul_nonneg_of_pos {X : Type*} [ordered_add_comm_monoid X] (m : ℕ)
  {x : X} (hx : 0 < x) : 0 ≤ m • x :=
begin
  induction m with m hm,
  { simp only [le_refl, zero_smul] },
  { rw [succ_nsmul x m],
    exact add_nonneg (le_of_lt hx) hm }
end
```
You might want to ask on zulip where a good home for this lemma is

Reply:
Will do!